etc/sql/oracle/schema.sql

```diff
@@ -1426,27 +1426,44 @@ CONSTRAINT DISTANCES_DEST_RSES_FK FOREIGN KEY(DEST_RSE_ID) REFERENCES RSES (ID),
 
 
 
--- 49 ) ========================================= RSE_TRANSFER_LIMITS table =========================================
+-- 49.1 ) ========================================= TRANSFER_LIMITS table =========================================
 
-  CREATE TABLE RSE_TRANSFER_LIMITS
-   (	RSE_ID RAW(16),
-	ACTIVITY VARCHAR2(50),
-	RSE_EXPRESSION VARCHAR2(3000),
-	MAX_TRANSFERS NUMBER(19,0),
-	TRANSFERS NUMBER(19,0),
-	WAITINGS NUMBER(19,0),
-        VOLUME NUMBER(19,0),
-	UPDATED_AT DATE,
-	CREATED_AT DATE,
+CREATE TABLE TRANSFER_LIMITS
+(
+    ID RAW(16),
+    RSE_EXPRESSION VARCHAR2(3000),
+    ACTIVITY VARCHAR2(50),
+    DIRECTION CHAR(1),
+    MAX_TRANSFERS NUMBER(19,0),
+    VOLUME NUMBER(19,0),
     DEADLINE NUMBER(19,0),
     STRATEGY VARCHAR2(25 CHAR),
-    DIRECTION VARCHAR2(25 CHAR),
-	 CONSTRAINT RSE_TRANSFER_LIMITS_PK PRIMARY KEY (RSE_ID, ACTIVITY),
-	 CONSTRAINT RSE_TRANSFER_LIMITS_RSE_ID_FK FOREIGN KEY(RSE_ID) REFERENCES RSES (ID),
-	 CONSTRAINT RSE_TRANSFER_LIMITS_CREATED_NN CHECK (CREATED_AT IS NOT NULL),
-	 CONSTRAINT RSE_TRANSFER_LIMITS_UPDATED_NN CHECK (UPDATED_AT IS NOT NULL)
-   ) ORGANIZATION INDEX COMPRESS 1;
+    TRANSFERS NUMBER(19,0),
+    WAITINGS NUMBER(19,0),
+    UPDATED_AT DATE,
+    CREATED_AT DATE,
+    CONSTRAINT TRANSFER_LIMITS_PK PRIMARY KEY (ID),
+    CONSTRAINT TRANSFER_LIMITS_RSE_EXPRESSION_NN CHECK (RSE_EXPRESSION IS NOT NULL),
+    CONSTRAINT TRANSFER_LIMITS_CREATED_NN CHECK (CREATED_AT IS NOT NULL),
+    CONSTRAINT TRANSFER_LIMITS_UPDATED_NN CHECK (UPDATED_AT IS NOT NULL)
+) ORGANIZATION INDEX COMPRESS 1;
+
+CREATE INDEX TRANSFER_LIMITS_SELECTORS_IDX ON TRANSFER_LIMITS (RSE_EXPRESSION, ACTIVITY) COMPRESS 1;
 
+-- 49.2 ) ========================================= RSE_TRANSFER_LIMITS table =========================================
+
+CREATE TABLE RSE_TRANSFER_LIMITS
```

Comment:
I thought this one is being dropped? In that case you can just take it out of the schema file.

Reply:
It is not being completely dropped, it is re-created with a different schema